unknown.go

```diff
@@ -52,7 +52,7 @@ func (UnknownNode) Commitment() *Point {
 }
 
 func (UnknownNode) GetProofItems(keylist) (*ProofElements, []byte, [][]byte, error) {
-	panic("trying to produce a commitment for a subtree missing from the stateless view")
+	return nil, nil, nil, errors.New("can't generate proof items for unknown node")
```

Comment:
This felt a bit aggressive, so I'm taking the same policy as we do in `HashedNode`.
LMK if we want to revert!